src/IECoreArnold/Renderer.cpp

```diff
@@ -2587,6 +2595,14 @@ class ArnoldLight : public ArnoldObjectBase
 				updateLightFilterLinks();
 			}
 
+			// Update mute state if needed.
+
+			bool newMuteState = m_attributes->muteLight();
+			if( AiNodeIsDisabled( m_lightShader->root() ) != newMuteState )
+			{
+				AiNodeSetDisabled( m_lightShader->root(), newMuteState );
+			}
```

Comment:
This crashes when `m_lightShader` is null, which it will be for lights designed for other renderers. To reproduce this, just make a Cycles disk light, and view it in the Arnold viewport. Unless you've seen evidence to the contrary, I'd assume we don't need the `AiNodeIsDisabled()` check here either - we don't perform the equivalent checks when setting other Arnold values.

Reply:
Fixed in 61a0719ffbda738276335ec4bfb8bfeb9476ab95